utils/getenv/getenv.go

```diff
@@ -0,0 +1,18 @@
+package getenv
+
+import (
+	"os"
+	"strconv"
+	"strings"
+)
+
+func parseEnvVar(env, defaultVal string) string {
+	if val, ok := os.LookupEnv(env); ok {
+		_, err := strconv.ParseBool(val)
+		if err == nil {
```

Comment:
This will not work in all cases. You can use a separate function for "boolean as a string" to make it easier. Also, you will need to add unit tests for this in a file called `utils_test.go`